powershell.html.markdown

```diff
@@ -554,6 +556,16 @@ True     False    Guitar                                   Instrument
 #>
 Get-Process | Foreach-Object ProcessName | Group-Object
 
+# Useful pipeline examples are iteration and filtering
+1..10 | ForEach-Object { "Loop number $PSITEM" }
+1..10 | where {$PSITEM -gt 5} | Format-Table
+
+# A noteable pitfall of the pipeline is it's performance when
+# compared with other options
+# additionally, raw bytes are not passed through the piipeline
```

Comment:
typo: piipeline

Reply:
Gotta love the Drop mech keyboard quality.